pkg/controllers/scheduling/scheduling_controller.go

```diff
@@ -147,28 +150,54 @@ func NewSchedulingController(
 		}, placementDecisionInformer.Informer()).
 		WithBareInformers(clusterInformer.Informer(), clusterSetInformer.Informer(), clusterSetBindingInformer.Informer()).
 		WithSync(c.sync).
+		ResyncEvery(ResyncInterval).
 		ToController(schedulingControllerName, recorder)
 }
 
 func (c *schedulingController) sync(ctx context.Context, syncCtx factory.SyncContext) error {
 	queueKey := syncCtx.QueueKey()
-	namespace, name, err := cache.SplitMetaNamespaceKey(queueKey)
-	if err != nil {
-		// ignore placement whose key is not in format: namespace/name
-		utilruntime.HandleError(err)
-		return nil
-	}
-
 	klog.V(4).Infof("Reconciling placement %q", queueKey)
-	placement, err := c.placementLister.Placements(namespace).Get(name)
-	if errors.IsNotFound(err) {
-		// no work if placement is deleted
+
+	if queueKey == "key" {
```

Comment:
Why is this being done instead of building a reverse index from AddOnPlacementScore to Placements?  Are you planning to build the reverse index later?  I could this as a starting point, but I wouldn't expect to do it this way regularly.

Additionally, you only need to do this if you have more than zero AddOnPlacementScores.  That check should be added so that only clusters using this alpha feature are impacted.

Reply:
Thanks for the comment, remove resync() from this PR, will create another PR to handle that.